### ISSUE TITLE
Clean up links to internal services

### ISSUE THREAD

**Issue by bonkey** [zalando-incubator/atlas-ios#121]
```
AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json:1
AtlasSDK/AtlasSDK/Options.swift:1
Documentation/Atlas/docsets/AtlasCheckout.docset/Contents/Resources/Documents/Enums.html:2
Documentation/Atlas/docsets/AtlasCheckout.docset/Contents/Resources/Documents/Global Variables.html:2
Documentation/Atlas/docsets/AtlasCheckout.docset/Contents/Resources/Documents/Extensions.html:2
Documentation/Atlas/docsets/AtlasCheckout.docset/Contents/Resources/Documents/index.html:4
Documentation/Atlas/docsets/AtlasCheckout.docset/Contents/Resources/Documents/Structs/Article.html:2
Documentation/Atlas/docsets/AtlasCheckout.docset/Contents/Resources/Documents/Structs.html:2
Documentation/Atlas/docsets/AtlasCheckout.docset/Contents/Resources/Documents/Typealiases.html:2
Documentation/Atlas/Enums.html:2
Documentation/Atlas/Extensions.html:2
Documentation/Atlas/Global Variables.html:2
Documentation/Atlas/index.html:4
Documentation/Atlas/Structs.html:2
Documentation/Atlas/Structs/Article.html:2
Documentation/Atlas/Typealiases.html:2
README.md:2
```

**Comment by bonkey** [comment]
[Safe method](https://help.github.com/articles/remove-sensitive-data/) recommended.

**Comment by artdaw** [comment]
GET*|config.json:1 will be automatically fixed in #112. All other sources are cleaned up.

**Comment by bonkey** [comment]
I found others:

```
AtlasMockAPI/AtlasMockAPI/JSON/POST*|checkouts.json
AtlasMockAPI/AtlasMockAPI/JSON/POST*|checkouts|CHECKOUT_ID.json
AtlasMockAPI/AtlasMockAPI/JSON/PUT*|checkouts|CHECKOUT_ID.json
```

And BTW. @artdaw have you used [safe method](https://help.github.com/articles/remove-sensitive-data/)?

**Comment by artdaw** [comment]
Unfortunately for docs I didn't

**Comment by bonkey** [comment]
Leftovers will be resolved in #267
